SPECS/moby-engine/moby-engine.spec

```diff
@@ -111,6 +112,9 @@ fi
 %{_unitdir}/*
 
 %changelog
+* Fri Aug 09 2024 Dallas Delaney <dadelan@microsoft.com> - 25.0.3-5
```

Comment:
There is a PR which is 2 weeks old for this:
- https://github.com/microsoft/azurelinux/pull/9953